waspc/data/Generator/templates/server/src/polyfill.ts

```diff
@@ -3,5 +3,8 @@
 
 import { webcrypto } from "node:crypto";
 
-// @ts-ignore
-globalThis.crypto = webcrypto as Crypto;
+if (typeof globalThis.crypto === "undefined") {
```

Comment:
The error message we see in `Node.js 20` is due to the `crypto` property already exists. This check ensures we don't try to set it if it exists.

Reply:
Something else I saw in the wild, check the Node.js version: https://github.com/mitschabaude/montgomery/blob/1b7c1d3f1adf7bdf39812fd23a5bdd5f95496171/src/extra/fix-webcrypto.ts#L3

Reply:
Checking node version is more indirect, because there is additional assumption that certain versions of don't have or don't have this functionality, while he could have just checked directly. Unless there is some other reasoning here, I prefer our solution.